CA_DataUploaderLib/IOconf/IOconfRow.cs

```diff
@@ -12,12 +12,14 @@ public IOconfRow(string row, int lineNum, string type)
             Type = type;
             LineNumber = lineNum;
             var list = ToList();
-            if (list[0] != Type) throw new Exception("IOconfRow: wrong format: " + Row);
+            if (list[0] != Type && type != "Unknown") throw new Exception("IOconfRow: wrong format: " + Row);
+            Name = list[1]; // could be overwritten elsewhere. 
```

Comment:
suggest to use this field in the UniqueKey + the field should be changed in the constructor of both IOconfOven and IOconfMap as in both cases list[1] is not the name.